src/equations/laplace_diffusion_2d.jl

```diff
@@ -1,17 +1,23 @@
 @doc raw"""
-    LaplaceDiffusion2D{T} <: AbstractParabolicEquations{2, 1}
+    LaplaceDiffusion2D(diffusivity, equations)
 
 `LaplaceDiffusion2D` represents a scalar diffusion term ``\nabla \cdot (\kappa\nabla u))``
-applied to each solution component.
+with diffusivity ``\kappa`` applied to each solution component defined by `equations`.
+
+If `equations` is not specified, a scalar `LaplaceDiffusion2D` is returned.
```

Comment:
```suggestion
with diffusivity ``\kappa`` applied to each solution component defined by `equations`.
```
I don't see this default value below